typescript/utils/src/altvm.ts

```diff
@@ -525,7 +525,11 @@ export interface ISigner<T = any, R extends ResSignAndBroadcast = any>
   extends IProvider<T> {
   getSignerAddress(): string;
 
-  signAndBroadcast(transactions: T[]): Promise<R>;
+  supportsMultiTransactions(): boolean;
+
+  sendAndConfirmTransaction(transaction: T): Promise<R>;
+
+  sendAndConfirmMultiTransactions(transactions: T[]): Promise<R>;
```

Comment:
is "multitransaction" cosmos lingo? when i see "multi" I tend to think "multisig"

perhaps consider "batching":
- `supportsTransactionBatching`
- `sendAndConfirmTransaction`
- `sendAndConfirmBatch`

or something like that

Reply:
probably the `batching` term, as it aligns with what we call it in agents/agent configs

Reply:
renamed multi to batch now